File: WebApiClient/common/src/ExerciseScoreModel.ts
export class ExerciseScoreModel
{
    public exerciseId: number = 0;
    public leftNumber:number = 0;
    public rightNumber:number = 0;
    public mathOperator: number = 0;
    public answer: string = '';
    public correctAnswerGiven: boolean = false;
    public userId: string = '';
    public level: number = 1;
    public highScore: number = 0;
}

File: WebApi/WebApi/Models/ViewModels/ExerciseScoreModel.cs
using System;
using WebApi.Enums;

namespace WebApi.Models.ViewModels
{
    public class ExerciseScoreModel
    {
        public int exerciseId { get; set; }
        public int leftNumber { get; set; }
        public int rightNumber { get; set; }
        public Operator mathOperator { get; set; }
        public double? answer { get; set; }
        public bool correctAnswerGiven { get; set; }
        public Guid userId { get; set; }
        public int level { get; set; }
        public int highScore { get; set; }
    }
}
